src/styles/main.scss

```diff
@@ -0,0 +1,55 @@
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 600px;
+  width: 600px;
+  margin: auto;
+  
+
+  &__arrow {
+    display: flex;
+    align-items: flex-end;
+    position: relative;
+    justify-content: center;
+
+    &-minute {
+      position: absolute;
+      width: 6px;
+      height: 150px;
+      border-radius: 3px;
+      background-color: #c71836;
+      animation: clock 3600s linear infinite;
+      transform-origin: bottom;
+    }
+
+    &-second {
+      position: absolute;
+      width: 6px;
+      height: 300px;
+      border-radius: 3px;
+      background-color: #0aa5ad;
+      animation: clock 60s linear infinite;
+      transform-origin: bottom;
+    }
+  }
```

Comment:
This looks convenient, but you should not split element names when apply nesting.
Use modifier instead.